app/src/test/java/com/nervousfish/nervousfish/util/CircularListTest.java

```diff
@@ -81,6 +81,7 @@ public void testEqualsHash() {
         list1.add(12);
         assertTrue(list3.equals(list4));
         assertTrue(list3.hashCode() == list4.hashCode());
+        assertFalse(list3.equals(null));
```

Comment:
You're already asserting that `list3` is equal to `list4`, and `list4` cannot be null here